devtools/test_dashboard/buttons.js

```diff
@@ -11,7 +13,7 @@ anchor.style.width = '1000px';
 
 function plotButtons(plots, figDir) {
 
-    Object.keys(plots).forEach( function (plotname) {
+    Object.keys(plots).forEach(function (plotname) {
```

Comment:
@etpinard (for me) just wanted to make sure I was understanding that rule. I'm okay with or without the commit for all `function ()` instances